Bugzilla/MFA/Duo.pm

```diff
@@ -65,23 +46,16 @@ sub prompt {
     -httponly => 1,
   );
 
-  print $cgi->header();
-  $template->process('mfa/duo/verify.html.tmpl', $vars)
-    || ThrowTemplateError($template->error());
+  my $username     = $self->property_get('user');
+  my $redirect_uri = $duo->create_auth_url($username, $token);
+
+  print $cgi->redirect($redirect_uri);
 }
 
+# Always return 0 (failure) in case check() for Duo called just to be safe
+# This should now happen.
```

Comment:
Should this be "This should not happen."? Any reason we can't just remove this?

Reply:
I will look into it as Bugzilla::MFA expects the downstream class to implement a check() method but I can also make it first make sure the method exists. And if not then return 0 by default.

Reply:
Why does the downstream class provide the check? What is it doing that we can always return false here knowing that would cause a failed check?